README.md

```diff
@@ -267,52 +269,79 @@ An HTTP GET request begins the IDV and KCC issuance flow.
 
 | Query Parameter              | Description                                                          | Required | References                                                                                                                                                                                                                    | Comments                                            |
 | :--------------------------- | :------------------------------------------------------------------- | :------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------- |
-| `presentation_definition_id` | The ID of a presentation definition describing the KCC to be issued. | n        | [Presentation&nbsp;Exchange&nbsp;2.0.0](https://identity.foundation/presentation-exchange/spec/v2.0.0/#presentation-definition) [tbDEX&nbsp;Offering](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#offering) | If not provided, the PFI chooses which KCC to issue |
+| `presentation_definition_id` | The ID of a presentation definition describing the KCC to be issued. | n        | [Presentation&nbsp;Exchange&nbsp;2.0.0](https://identity.foundation/presentation-exchange/spec/v2.0.0/#presentation-definition), [tbDEX&nbsp;Offering](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#offering) | If not provided, the PFI chooses which KCC to issue |
 
 ### Response
 
-The response is a [SIOPv2 Authorization Request](https://openid.github.io/SIOPv2/openid-connect-self-issued-v2-wg-draft.html#name-self-issued-openid-provider-a).
+The response is a [SIOPv2 Authorization Request](https://openid.github.io/SIOPv2/openid-connect-self-issued-v2-wg-draft.html#name-self-issued-openid-provider-a), 
+which is sent as a [JWT-Secured Authorization Request (JAR)](https://www.rfc-editor.org/rfc/rfc9101.html#name-authorization-request). 
+It may be returned as EITHER "request by value" or "request by reference".
```

Comment:
is there a way we can pick one to make this simpler to implement?

Reply:
Trying to say here that each KCC issuer should pick just one. PFI is picking by-value

Reply:
yes - I mean can we remove the optionality from the spec itself - no either - just you must do x

Reply:
ah ok. That's being discussed over in #33